### PR TITLE
CMS-643: Improve basic code styling

### PR DESCRIPTION
**Pull request by Flight**
**This pull request fixes the following issues**:
Updating the basic code styling (new lines, tabs, spaces, quotes).
It will slightly improve the readability and will improve the work speed in the future.

**Additional information**:
This PR improves the basic code readability, it doesn't fix any js/html issues.

### CODE REVIEW

**Comment by Brankub** [comment]
LGTM

File: filer/templates/admin/filer/widgets/admin_folder.html [hunk 1]
```diff
@@ -1,26 +1,29 @@
...
-    var plus = django.jQuery("#add_{{ id }}");
-    if (plus.length){
-        plus.remove();
-    }
-});
-</script>
+{% load i18n staticfiles %}
+
+{% spaceless %}
+    {% trans 'Folder' %}:
```
Comment (FinalAngel) [inline filer/templates/admin/filer/widgets/admin_folder.html]: trans should have double quotes

File: filer/templates/admin/filer/widgets/admin_folder.html [hunk 2]
```diff
@@ -1,26 +1,29 @@
...
+    <br>
+    {{ hidden_input }}
+    <script type="text/javascript">django.jQuery("#{{ id }}").hide();
+        django.jQuery("#{{ id }}_clear").click(function(){
+            django.jQuery("#{{ id }}").removeAttr("value");
+            django.jQuery("#{{ foldid }}").attr("src", "{% static noimg %}");
+            django.jQuery("#{{ descid }}").html("");
+        });
+        django.jQuery(document).ready(function(){
+            var plus = django.jQuery("#add_{{ id }}");
```
Comment (FinalAngel) [inline filer/templates/admin/filer/widgets/admin_folder.html]: single quotes

File: filer/templates/admin/filer/actions.html [hunk 3]
```diff
@@ -3,14 +3,14 @@
...
         <span class="action-counter">{{ selection_note }}</span>
         {% if paginator.count != paginated_items.object_list|length %}
-        <span class="all">{{ selection_note_all }}</span>
-        <span class="question">
-            <a href="javascript:;" title="{% trans "Click here to select the objects across all pages" %}">{% blocktrans with paginator.count as total_count %}Select all {{ total_count }} files and/or folders{% endblocktrans %}</a>
-        </span>
-        <span class="clear"><a href="javascript:;">{% trans "Clear selection" %}</a></span>
+            <span class="all">{{ selection_note_all }}</span>
+            <span class="question">
+                <a href="javascript:;" title="{% trans "Click here to select the objects across all pages" %}">{% blocktrans with paginator.count as total_count %}Select all {{ total_count }} files and/or folders{% endblocktrans %}</a>
```
Comment (FinalAngel) [inline filer/templates/admin/filer/actions.html]: single quotes here

**Comment by vxsx** [comment]
:+1:  on the assumption that there will be further improvements to eliminate mixed tabs / spaces etc

**Comment by FinalAngel** [comment]
LGTM

**Comment by FinalAngel** [comment]
@Flight can it be merged?

**Pull request closed without merging**